### ISSUE TITLE
Publish Nuget package

### ISSUE THREAD

**Issue by joecastro** [joecastro/wpf-shell#3]
Binaries should be published through Nuget.

**Comment by pgus** [comment]
Yes please. Looking at the old one at "https://www.nuget.org/packages/WPF.Shell.Integration.Library/" now. Would really appreciate if this updated was published.
